examples/missing_values.py

```diff
@@ -8,16 +8,22 @@
 Imputing does not always improve the predictions, so please check via cross-validation.
 Sometimes dropping rows or using marker values is more effective.
 
+In this example, we artificially mark some of the elements in complete
+dataset as missing. Then the score for the complete dataset, dataset without
```

Comment:
This is ungrammatical: there is no main verb in the sentence beginning "Then the score". It should say something like "Then we estimate performance using ...".